docs/framework/wcf/feature-details/client-configuration.md

```diff
@@ -35,7 +35,7 @@ You can use the Windows Communication Foundation (WCF) client configuration to s
               <servicePrincipalName value="host/localhost" />  
             </identity>  
           </endpoint>  
-// Add another endpoint by adding another <endpoint> element.  
+            // Add another endpoint by adding another <endpoint> element.  
```

Comment:
Shouldn't these in this file be changed to XML comments?
```suggestion
            <!-- Add another endpoint by adding another <endpoint> element. -->
```

Reply:
oh yes, thank you @Thraka